Documentation: note importer does not support zip files (#6209)
paperless-ngx/paperless-ngx: docs/administration.md

the `export` folder in your paperless source directory. Specify
`../export` as the `source`.

> If you have zipped the data and manifest 
> with the [Document exporter](#exporter) using the `-z` or `--zip` option, please
> extract the files inside the `export` folder first.
> 

Reviewer: ```suggestion
Note that .zip files (as can be generated from the exporter) are not supported.

```

Everything else here is extraneous, repeating the cli flags, mentioning the data and manifest, etc.

!!! note

    Importing from a previous version of Paperless may work, but for best
    results it is suggested to match the versions.
	

> !!! warning
>     The [Document importer](#importer) is meant to be run against a completely empty database
> 	and a set of empty paperless folders.
>     Unless you take great care in making sure that there will be no collisions
> 	(with the existing items in your paperless database) caused by the manifest data you
> 	place in the `export` folder, the document_importer script may crash on duplicate items/files.
> 	You run the risk of corrupting your target paperless instance if you import
> 	items into a paperless instance that is not completely empty. 

Reviewer: ```suggestion
!!! warning
    The importer should be run against a completely empty installation (database and directories) of Paperless-ngx.
```

Same theme as above.
